README.md

```diff
@@ -24,25 +23,55 @@ The simplest way to obtain the code is using the github .zip feature.
 
 Click [here](https://github.com/Aurorastation/Aurora.3/archive/master.zip) to get the latest stable code as a .zip file, then unzip it to wherever you want.
 
-The more complicated and easier to update method is using git.  You'll need to download git or some client from [here](http://git-scm.com/).  When that's installed, right click in any folder and click on "Git Bash".  When that opens, type in:
+If you wish to develop for/using this codebase, please read the "Development" section.
+
+### DEVELOPMENT
+To develop for/using this codebase, the following tools are recommended:
+
+- An updated and supported Windows Operating System version. (It might be possible to successfully follow the same general steps in other OSes, but they are untested)
+- [BYOND](https://www.byond.com/download/), with an active account ([Register](https://secure.byond.com/Join) if you do not have one already)
+- [Git](https://git-scm.com/downloads)
+- [Visual Studio Code (VSC)](https://code.visualstudio.com/download)
+- A GitHub account
+
+Prepare the development environment:
+
+1. Create a fork of this repository. Once logged in Github and while viewing this repository, click the "Fork" ribbon, or simply click [here](https://github.com/Aurorastation/Aurora.3/fork), give it a name and follow the forking instructions
+1. Download and install BYOND, login into it
+1. Download and install Git and VSC
+1. Create a folder in which you wish to keep the repository files, preferably on an SSD, then right click → "Git Bash Here". A new command prompt should appear.
+1. Inside the fork you have created above, click the "Code" ribbon and copy the URL under the HTTPS section
+1. Run the command `git clone URL_FROM_ABOVE`, changing the "URL_FROM_ABOVE" text with the URL you have copied before, and let it run to completion
+1. Open VSC and select "Open Folder", select the newly created folder
+1. Accept the prompts, install the extensions that are suggested, login to those who asks you to, reboot VSC as requested/needed
```

Comment:
```suggestion
- An updated and supported Windows operating system. (It might be possible to successfully follow the same general steps in other OSes, but they are untested.)
- [BYOND](https://www.byond.com/download), with an active account. ([Register](https://secure.byond.com/join) if you do not have one already.)
- [Git](https://git-scm.com/downloads)
- [Visual Studio Code](https://code.visualstudio.com/download) (aka VSC.)
- A GitHub account.

Prepare the development environment:
1. Create a fork of this repository. Once logged into GitHub and while viewing this repository, click the "Fork" ribbon, or simply click [here](https://github.com/Aurorastation/Aurora.3/fork). Give it a name and follow the forking instructions
2. Download and install BYOND. Login into it.
3. Download and install Git and VSC.
4. Create a folder where you wish to keep the repository files (preferably on an SSD), then right click and select "Git Bash Here". A new command prompt should appear.
5. Inside the fork you have created above, click the "Code" ribbon and copy the URL under the HTTPS section.
6. Run the command `git clone URL_FROM_ABOVE`, changing the "URL_FROM_ABOVE" text with the URL you have copied before, and let it run to completion.
7. Open VSC and select "Open Folder". Select the newly created folder.
8. Accept the prompts, install the extensions that are suggested, login to those who ask you to, and reboot VSC as needed.
```

Reply:
Not needed, the website auto-numbers them, as per https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax